docs/src/tutorials/yocto-linux.md

```diff
@@ -0,0 +1,150 @@
+`meta-tedge` layer. For more information, see the [getting started document on Yocto Project
+website](https://www.yoctoproject.org/software-overview/).
+
+
+`meta-tedge` is supported for **Yocto version 3.4 "Honister" and 4.0 "Kirkstone"**. Additionally, we will depend on `meta-openembedded`
+layer for Mosquitto.
+
+## Installation
+
+If you are not familiar with building Yocto distribution or you have not configured your build host yet, we strongly
+recommend to look into [official yocto documentation](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html)
+as the installation process will now skip all information that were mentioned there! For workspace organization or
+raspberry pi distribution, we also recommend this [guide](https://github.com/jynik/ready-set-yocto)
+
+> Most of the installation process is based on [Yocto Project Quick Build
+guide](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html).
+
+Before starting, be sure the host, where you plan to build a new Yocto image with thin-edge, meets the following
+requirements:
+
+- 50 Gbytes of free disk space
+- a Linux distributions that supports the Yocto Project, see the [Supported Linux
+Distributions](https://docs.yoctoproject.org/4.0.4/ref-manual/system-requirements.html#supported-linux-distributions)
+section in the Yocto Project Reference Manual. For detailed information on preparing your build host, see the [Preparing
+the Build Host](https://docs.yoctoproject.org/4.0.4/dev-manual/start.html#preparing-the-build-host) section in the Yocto
+Project Development Tasks Manual.
+- Git 1.8.3.1 or greater
+- tar 1.28 or greater
+- Python 3.6.0 or greater.
+- gcc 5.0 or greater.
+
+For the purposes of the tutorial, we assume `~/yocto-thinedge` working directory. If using other directory, be sure to
+change paths where needed.
+
+### Build Host Packages
+
+Install essential packages:
+
+```bash
+$ sudo apt install gawk wget git diffstat unzip texinfo gcc build-essential chrpath socat cpio python3 python3-pip python3-pexpect xz-utils debianutils iputils-ping python3-git python3-jinja2 libegl1-mesa libsdl1.2-dev pylint3 xterm python3-subunit mesa-common-dev zstd liblz4-tool
+```
+
+### Clone Poky Linux repository
+
+Clone the Poky Linux distribution sources. We'll be using version `kirkstone`. You can use `--depth=1` to speed up the
+process. If using these options, to see previous commits or other branches see
+[here](https://stackoverflow.com/questions/29270058/how-to-fetch-all-git-history-after-i-clone-the-repo-with-depth-1)
+and [here](https://stackoverflow.com/questions/17714159/how-do-i-undo-a-single-branch-clone).
+
+```
+$ git clone git://git.yoctoproject.org/poky --branch=kirkstone --depth=1
+```
+
+### Add layers
+
+We'll be using `meta-tedge` and `meta-openembedded`. First, fetch the repositories:
+
+```bash
+git clone https://github.com/thin-edge/meta-thin-edge
```

Comment:
I'm okay with name of the actual repo: https://github.com/thin-edge/meta-tedge.

Just need to be fixed here.